Makefile

```diff
@@ -81,7 +81,7 @@ docker-push: $(addsuffix -push,$(IMAGES))
 docker-push-latest-dev: $(addsuffix -push-latest-dev,$(IMAGES))
 
 %-push-latest-dev:
-	docker pull $(DOCKER_REGISTRY)/$*:$(PRESIDIO_LABEL)
+	docker pull $(DOCKER_REGISTRY)/$*:$(PRESIDIO_LABEL) ||:
```

Comment:
but this doesnt, It is not possible for a push scenario without a registry defined